docs/site/angular-auth-oidc-client/docs/documentation/configuration.md

```diff
@@ -5,29 +5,30 @@ sidebar_position: 2
 
 # Configuration
 
-You can pass three properties into the `AuthModule.forRoot(...)` method to configure the lib.
-
-```
-AuthModule.forRoot({
-  authWellKnown?: AuthWellKnownEndpoints;
-  config?: OpenIdConfiguration;
-  loader?: Provider;
-})
-```
+The lib has to be configured with the values of your choice. You can to that either with a static config or loading the config from an HTTP endpoint and map the values to the format the library expects.

```

Comment:
I would rephrase for clarity, like so:

Prior to using the library, you must configure it with the appropriate values for your environment. You can either configure the application statically, by providing the configuration values at design-time, or you can fetch the configuration from an HTTP endpoint. Configurations loaded from an HTTP endpoint must be mapped to the format the library expects.